src/main/scala/cromwell/binding/TaskInput.scala

```diff
@@ -0,0 +1,5 @@
+package cromwell.binding
+
+import cromwell.binding.types.WdlType
+
+case class TaskInput(name: String, types: Seq[WdlType], postfixQuantifier: Option[String])
```

Comment:
I'll add this as the comment for `types` in the ScalaDoc

> Because tasks define inputs that take 0-or-more and 1-or-more parameters (e.g. `${sep="," var*}`), the corresponding acceptable types would be `WdlString` and `Array[WdlString]`.  `WdlString` would effectively be treated as an `Array[WdlString]` of length 1.  When expanding this particular command line parameter, the items would be concatenated together using the separator in between them.  Allowing both `WdlString` and `Array[WdlString]` is a convenience mostly, so that client's don't have to specify arrays of size one.  If this ends up adding a lot of code complexity, then `types` can be changed to accept `WdlType` as its type without affecting too much